RecoTauTag/HLTProducers/src/HLTPFDiJetCorrCheckerWithDiTau.cc

```diff
@@ -68,22 +71,30 @@ void HLTPFDiJetCorrCheckerWithDiTau::produce(edm::StreamID iSId, edm::Event& iEv
 
   std::set<unsigned int> indices;
 
-  if (pfJets.size() > 1 && taus.size() > 1) {
-    for (unsigned int iJet1 = 0; iJet1 < pfJets.size(); iJet1++) {
-      for (unsigned int iJet2 = iJet1 + 1; iJet2 < pfJets.size(); iJet2++) {
-        bool correctComb = false;
-        const reco::PFJet& myPFJet1 = pfJets[iJet1];
-        const reco::PFJet& myPFJet2 = pfJets[iJet2];
+  for (unsigned int iJet1 = 0; iJet1 < pfJets.size(); iJet1++) {
+    for (unsigned int iJet2 = iJet1 + 1; iJet2 < pfJets.size(); iJet2++) {
+      bool correctComb = false;
+      const reco::PFJet& myPFJet1 = pfJets[iJet1];
+      const reco::PFJet& myPFJet2 = pfJets[iJet2];
+
+      if ((myPFJet1.p4() + myPFJet2.p4()).M() < mjjMin_)
+        continue;
 
-        if ((myPFJet1.p4() + myPFJet2.p4()).M() < mjjMin_)
+      for (unsigned int iTau1 = 0; iTau1 < taus.size(); iTau1++) {
+        if (reco::deltaR2(taus[iTau1]->p4(), myPFJet1.p4()) < dRmin2_)
+          continue;
+        if (reco::deltaR2(taus[iTau1]->p4(), myPFJet2.p4()) < dRmin2_)
           continue;
 
-        for (unsigned int iTau1 = 0; iTau1 < taus.size(); iTau1++) {
-          if (reco::deltaR2(taus[iTau1]->p4(), myPFJet1.p4()) < dRmin2_)
-            continue;
-          if (reco::deltaR2(taus[iTau1]->p4(), myPFJet2.p4()) < dRmin2_)
+        if (taus.size() == 1) {
+          if (taus[iTau1]->pt() < extraTauPtCut_)
             continue;
 
+          correctComb = true;
+          break;
+        }
+
+        else {
```

Comment:
Please remove the empty line between the `if` and the `else` blocks.

Reply:
So, I've never squashed/rebased before in git but I tried the following. 
git rebase -i HEAD~6
and set the first commit (the one at the top) to "pick" and the rest to "squash" in the interactive editor that popped up. 
However, it seems that my local changes were reverted to the first commit (so i should have squashed everything up to the last commit, and then changed that commit message). Somehow I am not able to pull the changes from this PR back (with git pull on the branch that I had been pushing from).

How do you suggest I proceed?

Reply:
I would do something like the following (most likely, there are much better ways):
```sh
cmsrel CMSSW_13_1_0_pre1
cd CMSSW_13_1_0_pre1/src
cmsenv
git cms-init
git cms-addpkg RecoTauTag/HLTProducers
git checkout my-cmssw/master-branch-extend-HLTModule-PFDiJetCorrChecker RecoTauTag/HLTProducers/src
git checkout -b master-branch-extend-HLTModule-PFDiJetCorrChecker

## check that the local changes are as intended
## fix manually the empty line as per the last comment

git add -u :/
git commit -m 'extended existing HLT tau-jet correlation module' 
scram b
scram b code-checks
scram b code-format
scram b
git push my-cmssw master-branch-extend-HLTModule-PFDiJetCorrChecker -f
```

Reply:
Thank you so much for the instructions. I think the lines 
```
git add -u :/
git commit -m 'extended existing HLT tau-jet correlation module' 
```
should go after the code-checks/formats, at least that's what I figured out just now since there was a code-formatting error that was fixed locally but not committed before the push. Everything should be resolved now, and also on the backport.

Reply:
You are right, my mistake. I edited my previous message for clarity.